spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc

```diff
@@ -459,7 +459,29 @@ When that happens, check the server log files for further details.
 [[dashboard-job-executions-steps-progress]]
 === Step Execution History
 
-Under *Step Execution History*, you can also view various metrics associated with the selected step, such as duration, read counts, write counts, and others.
+Under *Step Execution History*, you can also view various metrics associated with the selected step, such as duration, read counts, write counts, and others across all of its executions.
+For each metric there are 5 attributes that are available for each metric:
+
+* Count - The number of step executions that the metric could have participated.  It is not a count for the number of times the event occurred during each step execution.
+* Min - Shows the minimum value for this metric across all the step executions.
+* Max - Shows the maximum value for this metric across all the step executions.
+* Mean - Shows the mean value for this metric across all the step executions.
+* Max - Shows the standard deviation for this metric across all the step executions.
+
+NOTE: Counts (commit count for example) for a specific step execution, can be obtained by retrieving the job execution detail.
+
+The Step Execution contains the following metrics:
```

Comment:
Is `job execution detail`(line above)  the same as `step execution` ?

Reply:
Removing this line because this information is covered in another section.